views/overlays/commentators.hbs

```diff
@@ -9,75 +9,227 @@
     <script src="/socket.io/socket.io.js"></script>
 
     <!--TODO global-->
-    <script src="https://code.jquery.com/ui/1.12.0-beta.1/jquery-ui.min.js"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
     <link rel='stylesheet' href='/stylesheets/overlays/global-source.css' />
 
+    <!--Fouquet Depends-->
+    <meta charset="utf-8">
+      <meta http-equiv="x-ua-compatible" content="ie=edge">
+      <title></title>
+      <meta name="description" content="">
+      <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
`<meta>` tags, or at least the charset, are supposed to be right after the `<head>`, I believe https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/HTML5/Introduction_to_HTML5

Also the title element is included twice